test/functional/finalization_deposit.py

```diff
@@ -94,14 +94,15 @@ def test_successful_deposit(self, finalizer, proposer):
                    timeout=5)
 
         # mine a block to allow the deposit to get included
-        self.generate_sync(proposer)
+        self.generate_sync(proposer, nblocks=2)
```

Comment:
The comment states `a block`. Why was it necessary to move one block from the next generation call?